frontend/components/puzzle3.jsx

```diff
@@ -20,8 +19,8 @@ class Puzzle3 extends React.Component {
 
   handleClick(input) {
     return () => {
-      var tempArray = this.state.combo;
-      var flag = 'full';
+      const { combo: tempArray } = this.state;
```

Comment:
this sets tempArray to initial_combo?

I didn't know you can write it this way...hmm